api/graphql/opening_hours/opening_hours_types.py

```diff
@@ -1,7 +1,7 @@
 import datetime
+from datetime import timezone
```

Comment:
perhaps we should lean towards to `from django.utils import timezone`?

Reply:
Sure, I can give it a go 👍

Reply:
There might be better deprecation warnings etc 🤷

Reply:
Updated 👍  I also did some refactoring to remove repeating `get_default_timezone()` calls.